README.md

```diff
@@ -3,7 +3,7 @@
 **Tags:** permissions, section, access, acl, user management, custom roles, content editing, workflow, boston university, bu  
 **Requires at least:** 3.1  
 **Tested up to:** 4.5.3  
-**Stable tag:** 0.9.9  
+**Stable tag:** 0.10.0  
```

Comment:
Should be 0.1.0

Reply:
that doesn't seem right - from 9 to 1 in the same position? It looks like the tags start at 0.3.0 and go up - I almost made it 0.9.10 it such a small update

Reply:
Actually 0.1 is the first https://github.com/bu-ist/bu-section-editing/tree/6d01687032d76d2844367fb48bf8548f4b56db23

Reply:
Yeah. It wasn't done right from the start. I misread. Leave as 0.10.0